lib/awspec/generator/doc/type/acm.rb

```diff
@@ -0,0 +1,17 @@
+module Awspec::Generator
+  module Doc
+    module Type
+      class Acm < Base
+        def initialize
+          super
+          @type_name = 'Acm'
+          @type = Awspec::Type::Acm.new('example.com')
+          @ret = @type.resource_via_client
+          @matchers = []
+          @ignore_matchers = []
```

Comment:
Could you fix like https://github.com/k1LoW/awspec/blob/master/lib/awspec/generator/doc/type/ebs.rb#L13 ?